### PR TITLE
Add product variant csv import

### PR DESCRIPTION
**Pull request by eLBati**
This module adds a menu entry in
Sales > Configuration > Product Categories and attributes > Product Template CSV Import
that will work with the import of CSV file product.template.csv that contains variants.

### CODE REVIEW

File: product_variant_csv_import/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Product Variant CSV Import',
+    'version': '0.1',
```
Comment (tafaRU) [inline product_variant_csv_import/__openerp__.py]: Please update it according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: product_variant_csv_import/README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+==========================
+
+This module adds a menu entry in
+Sales > Configuration > Product Categories and attributes > Product Template CSV Import
+that will work with the import of CSV file product.template.csv that contains variants.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline product_variant_csv_import/README.rst:20]: Specify here the columns to be used in the CSV.

**Comment by archetipo** [comment]
:+1:

**Comment by jgrandguillaume** [comment]
Hi,

Thanks for the contrib, please add the OCA as author as well (as requested by the template https://github.com/OCA/maintainer-tools/tree/master/template/module)

Otherwise (and a part from other comments), I've tested it and it work well. Once you fixed those points, you get my :+1: 

Regards,

Joël

**Comment by eLBati** [comment]
@tafaRU @pedrobaeza @jgrandguillaume thanks, I made the suggested changes

**Comment by tafaRU** [comment]
Thanks :+1:

**Comment by alexis-via** [comment]
:+1:

**Comment by tafaRU** [comment]
This PR can be merged, could someone of @OCA/product-maintainers please taking care of this?

Thank you in advance.

**Comment by tafaRU** [comment]
Thanks, @rvalyi.

File: product_variant_csv_import/__init__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
```
Comment (elicoidal) [inline product_variant_csv_import/__init__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: product_variant_csv_import/README.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline product_variant_csv_import/README.rst:57]: https

File: product_variant_csv_import/README.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+==========================
+Product Variant CSV Import
+==========================
+
+This module adds a menu entry in
+
+Sales > Configuration > Product Categories and attributes > Product Template CSV Import
+
+that will work with the import of CSV file product.template.csv that contains variants.
```
Comment (elicoidal) [inline product_variant_csv_import/README.rst:13]: that will allow the import of variants via a product.template.csv CSV file.

File: product_variant_csv_import/README.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Usage
+=====
+
+See 'sample_csv' folder for examples: in product.template.csv, for every variant (product.product), you can use 'attribute_value_ids' field writing its values in the same way as any other many2many field
```
Comment (elicoidal) [inline product_variant_csv_import/README.rst:18]: Add a period

**Comment by elicoidal** [comment]
I added some details.
Besides we should avoid commercial logos in images.

**Pull request merged**